react-carousel-component/my-app/src/Container.js

```diff
@@ -0,0 +1,70 @@
+import { useState, useEffect } from 'react';
+import './Container.css'
+
+/**
+ * A container of items.
+ * One item is displayed at a time, with buttons to flip through them:
+ * - Next and Prev scroll forward and backward one item
+ * - An array of buttons scroll to a specific item
+ */
+export default function Container({ items }) {
+  const [current, setCurrent] = useState(0);
+
+  useEffect(() => {
+    const timerID = setInterval(() => {
+      setCurrent((current + 1) % items.length)
+    }, 3000);
+    return () => clearInterval(timerID)
+  } )
```

Comment:
`current` is a dependency but it's not listed. Though since this runs on every render, maybe it works OK. But it is resetting your interval on every render, which may not be strictly correct.